tests/dummy/app/pods/docs/guides/main/table-customization/template.md

```diff
@@ -42,6 +42,45 @@ lookup paths:
   {{demo.snippet name='table-customization-custom-cell-template.js' label='component.js'}}
 {{/docs-demo}}
 
+If you want to customize a header cell but still want to include the elements to sort and
+to resize a column, you can use the `ember-th/sort-indicator` and `ember-th/resize-handle`
+components:
+
+{{#docs-demo as |demo|}}
+  {{#demo.example}}
+    {{! BEGIN-SNIPPET docs-example-sortings.hbs }}
+    <div class="demo-container">
+      <EmberTable as |t|>
+        <t.head
+          @columns={{columns}}
+          @sorts={{sorts}}
+
+          @onUpdateSorts={{action (mut sorts)}}
+
+          @widthConstraint='gte-container'
+          @fillMode='first-column'
+        
+          as |h|
+        >
+          <h.row as |r|>
+            <r.cell as |columnValue columnMeta|>
+              {{ember-th/sort-indicator columnMeta=columnMeta}}
+              Custom Header {{columnValue.name}}
+              {{ember-th/resize-handle columnMeta=columnMeta}}
```

Comment:
These two `{{` invocations for the sort/resize components should probably also be using angle invocation.

```hbs
<EmberTh::SortIndicator @columnMeta={{columnMeta}} />
Custom Header {{columnValue.name}}
<EmberTh::ResizeHandle @columnMeta={{columnMeta}} />
```

Reply:
👍